packages/action-menu/test/index.ts

```diff
@@ -581,94 +579,86 @@ export const testActionMenu = (mode: 'sync' | 'async'): void => {
                 'initially selected item should maintain selection'
             ).to.be.true;
         });
-        it('allows top-level selection state to change', async () => {
```

Comment:
Revised this test into something more straightforward and easier to debug. See below...